<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actionButtons</class>
 <widget class="QWidget" name="actionButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="btView">
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #28a745;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 16px;
    font-weight: bold;
    width: 32px;
    height: 32px;
}

QToolButton:hover {
    background-color: #5cd65c;
}</string>
     </property>
     <property name="text">
      <string>View</string>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btEdit">
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #00008B;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 16px;
    font-weight: bold;
    width: 32px;
    height: 32px;
}

QToolButton:hover {
    background-color: #0000FF
}</string>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btDelete">
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #ff5100;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 16px;
    font-weight: bold;
    width: 32px;
    height: 32px;
}

QToolButton:hover {
    background-color: #ff5168;
}</string>
     </property>
     <property name="text">
      <string>Del</string>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
